terraform/memorystore.tf

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create the Memorystore (redis) instance
+resource "google_redis_instance" "redis-cart" {
+  name           = "redis-cart"
+  memory_size_gb = 1
+  region         = var.region_memorystore
```

Comment:
I would suggest we use the same region as the core app (defined as the "region" variable) for the Memorystore deployment. Having them separated does not provide enough illustrative value for our sample.